app/src/main/java/com/example/plantsapp/data/entity/RoomTask.kt

```diff
@@ -0,0 +1,45 @@
+package com.example.plantsapp.data.entity
+
+import android.net.Uri
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import com.example.plantsapp.domain.model.Task
+
+@Entity(tableName = "tasks")
```

Comment:
Also when you use some hardcoded fields for database in several times, it is not bad to move them to some object{}  with constants in it

For example object PlantsDatabaseContract { .... }

Maybe not every field, but some that are reused over different places